src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	public String getPrintableString();
```

Comment:
You have not implemented the second part of this activity, which was to print a list of printable objects.